### PR TITLE
Small fixes around iframe embeds

### PR DESCRIPTION
**Pull request by kesla**
Fixing iframe embeds:
- adding the fullScreen option to true for youtube embeds
- make fullscreen configurable for custom embeds
- Make width/height configurable for youtube
- Make size configurable for vine

R = @iefserge
T = minor

### CODE REVIEW

File: lib/types/youtube.js [hunk 1]
```diff
@@ -23,5 +23,7 @@ export const parse = ([elm]) => {
   };
 };
 
-export const render = ({youtubeId}) =>
-  <iframe src={`https://www.youtube.com/embed/${youtubeId}`} />;
+export const render = ({youtubeId, width = 640, height = 480}) => renderIframe({
```
Comment (iefserge) [inline lib/types/youtube.js]: youtube should be 16:9 right?

**Comment by iefserge** [comment]
LGTM

File: lib/types/youtube.js (hunk 1 above)
Comment (kesla) [inline lib/types/youtube.js]: Hm....

Now I learnt something new. Youtube can be different aspect ratio.

https://www.youtube.com/watch?v=dt-dDg6uod0 can be shared at `640 X 360` while https://www.youtube.com/watch?v=sUg401QLa7I can be shared at `640 X 480`.

I'd opt for `640 x 480` to be the default - but I'm interested in hearing your thoughts.

Comment (iefserge) [inline lib/types/youtube.js]: youtube.com uses 16:9, so I'd expect 16:9 to be default too. Probably not a big deal as it can be configured

Comment (kesla) [inline lib/types/youtube.js]: @iefserge that's just not true. The first example is 16:9 while the second example is 4:3. So youtube.com uses both.

Comment (kesla) [inline lib/types/youtube.js]: but yeah seem that 16:9 is more common than 4:3, I'll update the default width/height to `640 x 360`

Comment (iefserge) [inline lib/types/youtube.js]: @kesla I'm talking about the frame you see on youtube.com, frame is still 16:9, the video itself can have a different size, especially old videos from the old 4:3 days.

Comment (kesla) [inline lib/types/youtube.js]: Oh! Right! Smart thinking. Makes sense!

**Pull request merged**